EMBRACE_Client/LoginViewController.m

```diff
@@ -48,7 +70,29 @@ -(IBAction)login:(id)sender {
         //If student exists, pull up student information.
         //If student doesn't exist, create new student profile.
         //For the moment, assume student does not exist, and create a new student.
-        student = [[Student alloc] initWithName:firstName :lastName];
+        
+        NSString* documentsPath = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) objectAtIndex:0];
+        NSString* tempFileName = [NSString stringWithFormat:@"%@ %@ %@.txt",school, firstName, lastName];
+        NSString* doesFileExist = [documentsPath stringByAppendingPathComponent:tempFileName];
+        BOOL fileExists = [[NSFileManager defaultManager] fileExistsAtPath:doesFileExist];
+        
+        if(fileExists)
```

Comment:
Might want to add more comments about what is going on when the file already exists.